domains/ryannkim327.json

```diff
@@ -0,0 +1,13 @@
+{
+    "description": "Ryann Kim Sesgundo, a student from Dalubhasaan ng Lungsod ng Lucena",
+    "repo": "https://github.com/RyannKim327/ryannkim327.github.io",
+    "owner": {
+        "username": "RyannKim327",
+        "email": "weryses19@gmail.com",
+        "twitter": ""
```

Comment:
```suggestion
```
Remove it if you are going to leave it empty

Reply:
i thought it is required, but I commit the suggestion so that it was already removed, thanks btw.